articles/portal/ptl-ref-portal-api.md

```diff
@@ -212,593 +212,45 @@ Returns a response with a list of up to 10 VMs
 ```
 
 ## GET /api/my_calls
-
+#### With the move to the ITSM tool this functionality is no longer available.
```

Comment:
Could you make these messages not headers?

```suggestion
With the move to the ITSM tool this functionality is no longer available.
```

If you want them to standout, you can use bold

```suggestion
**With the move to the ITSM tool this functionality is no longer available.**
```

Reply:
It might be a better idea to use alerts, as I've mentioned [here](https://github.com/UKCloud/documentation/pull/607#discussion_r300638996).

Reply:
Alerts look like a grand idea, i'll incorporate them in